config.json

```diff
@@ -952,6 +952,37 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "icon": "features-oop",
+      "title": "Modern",
+      "content": "Delphi is a modern, evolving language."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Cross-platform",
+      "content": "Delphi runs on almost any platform and chipset."
+    },
+    {
+      "icon": "features-functional",
+      "title": "Multi-paradigm",
+      "content": "Delphi is primarily an object-oriented language, but also has lots of functional features."
+    },
+    {
+      "icon": "features-lazy",
+      "title": "General purpose",
+      "content": "Delphi can be used for a wide variety of workloads, like websites, console applications, and even games."
```

Comment:
This line slightly exceeds the maximum of 100 characters. Note that you _can_ merge this and fix later, as configlet does not yet have this check built in. Up to you :)